tests/token/transfer.act

```diff
@@ -7,18 +7,18 @@ iff
    value <= balanceOf[CALLER]
    CALLER =/= to => balanceOf[to] + value < 2^256
 
-storage
-
 case CALLER =/= to:
 
-   balanceOf[CALLER] => balanceOf[CALLER] - value
-   balanceOf[to]     => balanceOf[to] + value
+   storage
```

Comment:
Wouldn't this be confused with the storage layout of the contract?

Reply:
it seems different in form enough to me to not cause any confusion. But calling it `state` would also alleviate this concern